.github/workflows/test.yml

```diff
@@ -39,10 +39,20 @@ jobs:
         name: setup terraform
```

Comment:
Looks like test.yml is not getting triggered as part of github actions

Reply:
test.yml is getting called now.